repo/predacons.py

```
@@ -42,13 +42,19 @@ async def load_model(model_name:str):
             draft_model_name=draft_model_name
         ) 
-          
-        # tokenizers = AutoTokenizer.from_pretrained(path)
-        tokenizers = predacons.load_tokenizer(path) #Updated
+        
+        # Check if processor should be used
+        use_processor = str2bool(os.getenv(model_name + "_use_processor"))
+        processor = None
+        tokenizers = None
+        if use_processor:
+            processor = predacons.load_processor(path)
```

Pylint (3.3.7)

[error] Module 'predacons' has no 'load_processor' member

(E1101)